naslib/defaults/trainer.py

```diff
@@ -108,11 +113,26 @@ def search(self, resume_from="", summary_writer=None, after_epoch: Callable[[int
 
         for e in range(start_epoch, self.epochs):
 
+            # create the arch directory (without overwriting)
+            if self.config.save_arch_weights: 
```

Comment:
It is better for readability to write this as `if self.config.save_arch_weights is True`